docs/config.md

```diff
@@ -26,6 +26,9 @@ The explanation and default value of each configuration item are as follows:
   # enable_selinux indicates to enable the selinux support.
   enable_selinux = false
 
+  # runtime_option_pod_annotations is list of pod annotations passed to OCI annotations
+  runtime_option_pod_annotations = []
```

Comment:
Can we make this a per runtime option? As suggested in https://github.com/containerd/cri/pull/1077#issuecomment-470626651

Reply:
done